pages/common/vegeta.md

```diff
@@ -0,0 +1,20 @@
+# vegeta
+
+> HTTP load testing tool.
+> Vegeta is a versatile HTTP load testing tool built out of a need to drill HTTP services with a constant request rate. It can be used both as a command line utility and a library.
+
+- Launch an attack during 30 seconds:
+
+`echo "GET http://{{adress/to/the/resource}}" | vegeta attack -duration=30s`
+
+- Launch an attack and ask for a report:
+
+`echo "GET http://{{adress/to/the/resource}}" | vegeta attack -duration=30s | vegeta report`
+
+- Launch an attack and plot the results (latency/time):
+
+`echo "GET http://{{adress/to/the/resource}}" | vegeta attack -rate=10 -duration=30s | vegeta plot > {path/to/output}/vegeta-results.html`
+
+- Launch multiple URL attack with a file input:
```

Comment:
```suggestion
- Launch an attack against multiple URLs from a file:
```

Perhaps this will read better?

Also, could `requests.txt` be something like `{{path/to/request_urls.txt}}` please?